content/news/052/index.md

```diff
@@ -70,6 +71,31 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Untitled Pixel Wizards Game][pixel-wizards]
+<video controls autoplay muted loop> <source type="video/mp4" src="untitled-pixel-wizards-game.mp4" onerror="parentNode.parentElement.innerText = 'Sorry, your browser does not support the video codec. Try a different browser!'"> />
```

Comment:
In my zola setup I have a shortcode for video embeds to avoid repeating the relevant HTML - let me know if you'd like me to add one here.

Re chosen attributes:

* I think `autoplay` is desirable for this "gif replacement" use case. You could add some further machinery to display some affordance that encourages readers to click on videos to play them, but autoplay seems simple & easy.
*  `muted` is necessary to make `autoplay` work these days.. thank god.
* `loop` is pretty much necessary with autoplay IMO, otherwise most of the videos will be missed
* `controls` is good so folks can pause & skip backwards/forwards, but some folks will have autoplay disabled at the browser level (I did for a while in Firefox) so it's good to show controls for them so they can more easily tell that it's not a static image.

Reply:
Okay I got a lint failure ` content/news/052/index.md:75:1 MD033/no-inline-html Inline HTML [Element: video]` so I've added the shortcode now.

It's neater this way anyway, and a bit more robust (since the caption styling for images/videos appears to be susceptible to breaking due to extra elements being introduced in between the image/video and the caption).

Reply:
Yeah, shortcode is better. Thanks :)